tools/generate-files.js

```diff
@@ -1,9 +1,13 @@
 var generator = require('../dist/sfdx-generator.cjs')
```

Comment:
Use const whenever possible instead of var while at it. If not possible, use let. If still not then use var.
(TL;DR: const>let>var)